model/k8s/metrics.yaml

```
@@ -469,3 +469,20 @@ groups:
     attributes:
       - ref: k8s.namespace.phase
         requirement_level: required
+
+  # k8s.container.* metrics
+  - id: metric.k8s.container.ready
+    type: metric
+    metric_name: k8s.container.ready
+    stability: development
+    brief: >
+      Specifies whether the container is currently passing its readiness check,
+      and is ready to accept traffic (0 for no, 1 for yes).
+    instrument: updowncounter
+    unit: "{container}"
+    note: |
+      This metric aligns with the `ready` field of the
+      [K8s ContainerStatus](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#containerstatus-v1-core).
```

can we be more precise than 'aligns' ?

E.g. 
```suggestion
      This metric SHOULD reflect the value of the `ready` field in the
      [K8s ContainerStatus](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#containerstatus-v1-core).
```

---

fixed